python/ccxt/base/exchange.py

```diff
@@ -1622,6 +1622,7 @@ def parse_orders(self, orders, market=None, since=None, limit=None, params={}):
 
     def safe_currency_code(self, currency_id, currency=None):
         code = None
+        currency_id = str(currency_id)
```

Comment:
```Python
mbp:ccxt.pro igorkroitor$ python3
Python 3.7.2 (v3.7.2:9a3ffc0492, Dec 24 2018, 02:44:43)
[Clang 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> str(None)
'None'
```

Reply:
I think we should swap the outer `if`s, what do you say?

Reply:
Great thinking, thank you!

Reply:
Thx to you for the fix, merging shortly! )